### PR TITLE
Fix Possible Datarace

### PR DESCRIPTION
**Pull request by azooz2003-bit**

### CODE REVIEW

File: CanvasPlusPlayground/CourseAnnouncementManager.swift [hunk 1]
```diff
@@ -26,14 +26,6 @@ import Foundation
                 self.announcements = cached.sorted(by: { 
                     ($0.createdAt ?? Date()) > ($1.createdAt ?? Date())
                 })
-            },
```
Comment (rahulon12) [inline CanvasPlusPlayground/CourseAnnouncementManager.swift:29]: Do we want to remove this?

**Review by rahulon12 — CHANGES_REQUESTED** [review]

File: CanvasPlusPlayground/CourseAnnouncementManager.swift (hunk 1 above)
Comment (rahulon12) [reply CanvasPlusPlayground/CourseAnnouncementManager.swift:29]: I mean the `onNewBatch` closure that you removed. I thought that was only for testing purposes.

Comment (azooz2003-bit) [reply CanvasPlusPlayground/CourseAnnouncementManager.swift:29]: It's trivial, safer to omit for now.

**Review by rahulon12 — APPROVED** [review]

File: CanvasPlusPlayground/CourseAnnouncementManager.swift (hunk 1 above)
Comment (rahulon12) [reply CanvasPlusPlayground/CourseAnnouncementManager.swift:29]: OK, but I'm still using it for people

**Pull request merged**